daprdocs/content/en/reference/components-reference/supported-state-stores/setup-azure-blobstorage.md

```diff
@@ -32,21 +34,32 @@ spec:
 The above example uses secrets as plain strings. It is recommended to use a secret store for the secrets as described [here]({{< ref component-secrets.md >}}).
 {{% /alert %}}
 
+## Versioning
+
+Dapr has 2 versions of the Azure Blob Storage state store component: `v1` and `v2`. It is recommended to use `v2`, as `v1` is deprecated.
```

Comment:
Can you add it to this list https://docs.dapr.io/operations/support/breaking-changes-and-deprecations/. Presumably this is the annoucement of the deprecation?

Reply:
No we are not going to remove v1, also because there's no upgrade path. Although that's now legacy.

Reply:
Ok. Then let's not use the term deprecates, since this has a specific meaning that this will be removed in 2 releases. 
Suggest changing this to "It is recommended to use `v2`, as `v1` is considered legacy".

Reply:
I'm ok with that

```suggestion
Dapr has 2 versions of the Azure Blob Storage state store component: `v1` and `v2`. It is recommended to use `v2` for all new applications. `v1` is considered legacy and is preserved for compatibility with existing applications only.
```